### PR TITLE
LOC -> PACKAGE_PIN in generate_constraint.py

### PR DESCRIPTION
**Pull request by hanw**
https://forums.xilinx.com/t5/Vivado-TCL-Community/PACKAGE-PIN-or-LOC/td-p/325765

### CODE REVIEW

**Comment by hanw** [comment]
And "name" need a 'u' in front.

File: scripts/generate-constraints.py [hunk 1]
```diff
@@ -108,9 +108,9 @@ def newArgparser():
                 pinInfo = copy.copy(boardPinInfo[pinName])
             else:
                 print('Missing pin description for', pin, pinName, projectPinInfo, file=sys.stderr)
-                pinInfo['LOC'] = 'fmc.%s' % (pinName)
+                pinInfo['PACKAGE_PIN'] = 'fmc.%s' % (pinName)
                 errorDetected = True
-            pinInfo['name'] = pin
+            pinInfo[u'name'] = pin
```
Comment (jameyhicks) [inline scripts/generate-constraints.py]: why is this the only unicode string?

Comment (hanw) [inline scripts/generate-constraints.py]: other keys in pinInfo is already unicode, as far as I can tell from debugging message.

Comment (jameyhicks) [inline scripts/generate-constraints.py]: OK. That is because it is read in as JSON. Seems reasonable to keep them consistent.

**Pull request merged**